_posts/2020-09-26-geoserver-2-18-0-released.md

```diff
@@ -0,0 +1,292 @@
+---
+author: ianturton
+comments: false
+date: 2020-09-26
+layout: post
+title: GeoServer 2.18-0 Released
+categories:
+- Announcements
+tags:
+- Release
+---
+
+The GeoServer community is pleased to share the availability
+of [GeoServer 2.18.0](http://geoserver.org/release/2.18.0/)
+Release. Downloads are available
+([zip](https://sourceforge.net/projects/geoserver/files/GeoServer/2.18.0/geoserver-2.18.0-bin.zip/download)
+and
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.18.0/geoserver-2.18.0-war.zip/download))
+along with
+[documentation](https://sourceforge.net/projects/geoserver/files/GeoServer/2.18.0/geoserver-2.18.0-htmldoc.zip/download)
+and more than 40
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.18.0/extensions/).
+
+This GeoServer release is made in conjunction with [GeoTools 24.0](https://geotoolsnews.blogspot.com/2020/09/geotools-240-released.html)
+and GeoWebCache 1.18.0.
+
+We would like to thank everyone who contributed to this release, and helped with testing it.
+
+Thanks to Jukka Rahkonen, Jody Garnett and Brad Hards for initial testing of this release. This release was carried out by Ian Turton of Astun Technology and the support of the UK Hydrographic Office.
+
+### Black Lives Matter
+
+The casual use of the word "slave" in computer software is an unnecessary
+reference to a painful human experience that continue to impact society.
+
+  * Change to the use of primary / replica in description of GeoServer
+  [clustering](https://docs.geoserver.org/latest/en/user/community/jms-cluster/activemq/JDBC.html)
```

Comment:
It seem some of these pages still have master / slave in the diagrams.

Can we link to https://docs.geoserver.org/latest/en/user/community/jms-cluster/index.html (which has fixed diagrams).